### PR TITLE
Feature/device status

### PR DESCRIPTION
**Pull request by DaQuirm**
Use Household API to get device status and introduce the `Unknown` status :cat: 
@larsblumberg @ghophp

### CODE REVIEW

File: public/app/common/components/device-status/device-status.cjsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+React            = require 'react'
```
Comment (DaQuirm) [inline public/app/common/components/device-status/device-status.cjsx]: :coffee:

File: public/app/common/components/device-status/device-status-text/style.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @import "components-theme";
 
-.cc-device-state-text {
```
Comment (larsblumberg) [inline public/app/common/components/device-status/device-status-text/style.scss]: Wouldn't the correct wording be _state_? See also http://english.stackexchange.com/questions/12958/status-vs-state

File: public/app/common/components/icon/icon.cjsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    styleMixin require('./style.scss')
+    PureRenderMixin
+  ]
+
+  getIconTypeClass: ->
+    if @props.type?
+      "cc-icon-#{@props.type}"
+    else
+      ''
+  render: ->
```
Comment (larsblumberg) [inline public/app/common/components/icon/icon.cjsx]: Add an empty line before `render:`

File: public/app/common/components/icon/style.scss [hunk 4]
```diff
@@ -29,6 +29,10 @@
     background-image: url('./images/state-disconnected.svg');
```
Comment (larsblumberg) [inline public/app/common/components/icon/style.scss]: You renamed _state_ to _status_. +1 for consistent naming. You might want to rename images also.

**Comment by larsblumberg** [comment]
Besides the comments, LGTM.

I couldn't however see, where the text _unknown_ is printed in the device details in case the device has such a status.

**Pull request merged**